### DISCUSSION TITLE
Sort problems; row multi-select

### DISCUSSION THREAD

**Discussion by darkmayuk** [AntlerVC/firetable#1297] [Support / Q&A] [answered]
Hello - I'm a new Rowy user. I like it so far. But I have two issues:

1. The sort function seems broken. Am I missing something? I have a table with about 20 rows. I select a column, and sort ascending (just as you would on a spreadsheet). Rowy acts like it's doing something ... and then returns the table in exactly the same state as it was, not sorted at all by the desired column. Strange.
2. Is it possible to select more than one row, and then edit all selected rows at the same time? If you're trying to make the same change to (say) 100 rows, it's painful to have to go through one by one and change them. Again, I hope I'm missing something here.

Thanks for your help.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @darkmayuk, we're currently trying to investigate this sorting/filtering issue. It;d be really helpful if you could share a recording of the issue via https://www.jam.dev?

For bulk updating the fields, you could utilize the import via CSV option. To know more about how you can utilize import via CSV for bulk updating, check out our detailed docs: https://docs.rowy.io/import-export-data/import-csv#how-to-bulk-update-if-you-have-autogenerated-ids.

**Reply by darkmayuk** [reply]
Thanks for your reply. Regarding sorting, see the jam video here: https://jam.dev/c/fdfc5603-d047-4e69-955d-8786611d1f3f

Regarding bulk updating, that's a good suggestion, I'll use that.
